packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -269,10 +269,17 @@ const FILTERS = {
 @addSubscriptions({
   remotes: cb =>
     subscribeRemotes(rawRemotes => {
-      rawRemotes = map(rawRemotes, remote => ({
-        ...remote,
-        ...parse(remote.url),
-      }))
+      rawRemotes = map(rawRemotes, remote => {
+        let parsedUrl
+        try {
+          parsedUrl = parse(remote.url)
+        } catch (_) {}
```

Comment:
Maybe we should log this error, so we know were to look if the remote appears broken?

Reply:
You're right :+1: